src/main/java/org/elasticsearch/common/joda/Joda.java

```diff
@@ -135,9 +137,9 @@ public static FormatDateTimeFormatter forPattern(String input, Locale locale) {
         } else if ("yearMonthDay".equals(input) || "year_month_day".equals(input)) {
             formatter = ISODateTimeFormat.yearMonthDay();
         } else if ("epoch_second".equals(input)) {
```

Comment:
This should probably be `epoch_seconds` to match the fact the `epoch_millis` is plural